04_numeronym/saurabhvyas9/main_test.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	want := strconv.Quote("[a11y K8s abc]\n")
+	actual := strconv.Quote(buf.String())
+	assert.Equalf(t, want, actual, "Numeronym failed.")
+}
+
+var testCases = map[string]struct {
+	input,
+	want []string
+}{
+	"main":               {input: []string{"accessibility", "Kubernetes", "abc"}, want: []string{"a11y", "K8s", "abc"}},
+	"Small strings":      {input: []string{"x", "yz", "xyz", "wxyz"}, want: []string{"x", "yz", "xyz", "w2z"}},
+	"String lines":       {input: []string{"Have fun", "Go Lang"}, want: []string{"H6n", "G5g"}},
+	"Alphanumeric":       {input: []string{"12as34sdf5678asd90", "abcd232ef"}, want: []string{"1160", "a7f"}},
+	"Empty":              {input: []string{"", ""}, want: []string{"", ""}},
+	"Special Characters": {input: []string{"%*(*)ASDjahs(", "absd*?>^&%$asd"}, want: []string{"%11(", "a12d"}},
```

Comment:
Include non-ascii unicode as well (e.g. "übernächtig")

Reply:
Rejected. 
Unicode will not work with the logic defined in my main file. 
Conversion to rune type would be required. Didn't update that much, hope that is alright.

Reply:
Thanks for advising this. 
I didn't earlier think of it but looked it up and found a blog for string, bytes, runes and characters. Good to know.

Reply:
Included test case and support in main file for non-ascii unicode runes.

Reply:
Done.